apps/taxes/src/pages/api/webhooks/order-fully-refunded.ts

```diff
@@ -0,0 +1,60 @@
+import { SaleorAsyncWebhook } from "@saleor/app-sdk/handlers/next";
+
+import {
+  OrderFullyRefundedEventSubscriptionFragment,
+  UntypedOrderFullyRefundedSubscriptionDocument,
+} from "../../../../generated/graphql";
+import { saleorApp } from "../../../../saleor-app";
+
+import { WebhookResponse } from "../../../modules/app/webhook-response";
+import { getActiveConnectionService } from "../../../modules/taxes/get-active-connection-service";
+import { createLogger } from "../../../logger";
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+export type OrderFullyRefundedPayload = Extract<
+  OrderFullyRefundedEventSubscriptionFragment,
+  { __typename: "OrderFullyRefunded" }
+>;
+
+export const orderFullyRefundedAsyncWebhook = new SaleorAsyncWebhook<OrderFullyRefundedPayload>({
+  name: "OrderRefunded",
+  apl: saleorApp.apl,
+  event: "ORDER_FULLY_REFUNDED",
+  query: UntypedOrderFullyRefundedSubscriptionDocument,
+  webhookPath: "/api/webhooks/order-fully-refunded",
+});
+
+export default orderFullyRefundedAsyncWebhook.createHandler(async (req, res, ctx) => {
+  const logger = createLogger(ctx.event);
+  const { payload } = ctx;
+  const webhookResponse = new WebhookResponse(res);
+
+  logger.debug("Handler called with payload");
+
+  try {
+    const order = payload.order;
+
+    if (!order) {
+      throw new Error("Insufficient order data");
+    }
+
+    const appMetadata = payload.recipient?.privateMetadata ?? [];
+    const channelSlug = order.channel.slug;
+    const taxProvider = getActiveConnectionService(channelSlug, appMetadata, ctx.authData);
+
+    logger.info("Refunding order...");
```

Comment:
I think if we log something like this we should include the payload:

```suggestion
    logger.info({ order: payload }, "Refunding order");
```